tests/integration/tty.bats

```diff
@@ -116,3 +116,67 @@ function teardown() {
 	[[ ${lines[0]} =~ 1000 ]]
 	[[ ${lines[1]} =~ 5 ]]
 }
+
+@test "runc exec [tty consolesize]" {
+	# allow writing to filesystem
+	sed -i 's/"readonly": true/"readonly": false/' config.json
+
+	# run busybox detached
+	runc run -d --console-socket $CONSOLE_SOCKET test_busybox
+	[ "$status" -eq 0 ]
+
+	# check state
+	wait_for_container 15 1 test_busybox
+
+	# make sure we're running
+	testcontainer test_busybox running
+
+# write a process.json that prints tty info to a file
+cat <<EOF > $BATS_TMPDIR/write-tty-info.json
+{
+	"terminal": true,
+	"consoleSize": {
+		"height": 10,
+		"width": 110
+	},
+	"args": [
+		"/bin/sh",
+		"-c",
+		"/bin/stty -a > $BATS_TMPDIR/tty-info"
```

Comment:
`$BATS_TMPDIR` is on the host not in the container. Since it's generally `/tmp` this won't cause obvious issues, but just use `/tmp` inside the container.